app/src/pages/admin/projectsPage/projectsPage.jsx

```diff
@@ -111,12 +120,12 @@ export class ProjectsPage extends Component {
       },
     ];
 
-    if (projectId) {
+    if (projectKey) {
       breadcrumbs.push({
-        title: projectId,
+        title: `${projectKey}`,
```

Comment:
It is string by default as described in the props.

Reply:
Done.